src/integration-test/java/uk/gov/ida/integrationtest/DefaultTrustStoreTest.java

```diff
@@ -0,0 +1,158 @@
+package uk.gov.ida.integrationtest;
+
+import io.dropwizard.testing.ConfigOverride;
+import org.junit.ClassRule;
+import org.junit.Test;
+import uk.gov.ida.integrationtest.helpers.MatchingServiceAdapterAppRule;
+import uk.gov.ida.saml.metadata.EidasMetadataConfiguration;
+import uk.gov.ida.saml.metadata.MetadataResolverConfiguration;
+
+import java.net.URI;
+import java.security.KeyStoreException;
+import java.util.ArrayList;
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.ida.matchingserviceadapter.configuration.ConfigurationConstants.INTEGRATION_TRUST_ANCHOR_URI;
+import static uk.gov.ida.matchingserviceadapter.configuration.ConfigurationConstants.INTEGRATION_EIDAS_METADATA_SOURCE_URI;
+import static uk.gov.ida.matchingserviceadapter.configuration.ConfigurationConstants.PRODUCTION_EIDAS_METADATA_SOURCE_URI;
+import static uk.gov.ida.matchingserviceadapter.configuration.ConfigurationConstants.PRODUCTION_TRUST_ANCHOR_URI;
+
+public class DefaultTrustStoreTest {
+
+    @ClassRule
+    public static final MatchingServiceAdapterAppRule prodMsaWithDefaultTruststoresApplicationRule = new MatchingServiceAdapterAppRule(
+            true,
+            "verify-matching-service-adapter-default-truststores.yml",
+            false,
+            ConfigOverride.config("metadata.environment", "PRODUCTION"),
+            ConfigOverride.config("europeanIdentity.aggregatedMetadata.environment", "PRODUCTION")
+    );
+
+    @ClassRule
+    public static final MatchingServiceAdapterAppRule integrationMsaWithDefaultTruststoresApplicationRule = new MatchingServiceAdapterAppRule(
+            true,
+            "verify-matching-service-adapter-default-truststores.yml",
+            false,
+            ConfigOverride.config("metadata.environment", "INTEGRATION"),
+            ConfigOverride.config("europeanIdentity.aggregatedMetadata.environment", "INTEGRATION")
+    );
+
+    @Test
+    public void prodTruststoresShouldContainCorrectCertificates() throws KeyStoreException {
+        MetadataResolverConfiguration metadataConfiguration = prodMsaWithDefaultTruststoresApplicationRule
+                .getConfiguration()
+                .getMetadataConfiguration()
+                .orElseThrow();
+        ArrayList<String> hubAliases = Collections.list(metadataConfiguration.getHubTrustStore().orElseThrow().aliases());
```

Comment:
Just me, but I'd prefer the assignment to be to the `List` interface, rather than the `ArrayList` implementation